client/lib/wp/sync-handler/index.js

```diff
@@ -32,7 +72,7 @@ export class SyncHandler {
 	 */
 	constructor( handler ) {
 		// expose `syncHandler` global var (dev mode)
-		if ( 'development' === config( 'env' ) ) {
+		if ( 'development' === config( 'env' ) && typeof window !== 'undefined' ) {
```

Comment:
> Avoiding server-side here ?

No, I'm just avoiding a warning you would otherwise get when running tests locally. I'd rather make this conditional than add `window` to the testing environment.